internal/service/timestreaminfluxdb/db_cluster.go

```diff
@@ -136,12 +141,19 @@ func (r *dbClusterResource) Schema(ctx context.Context, req resource.SchemaReque
 				CustomType: fwtypes.StringEnumType[awstypes.ClusterDeploymentType](),
 				Optional:   true,
 				Computed:   true,
-				Default:    stringdefault.StaticString(string(awstypes.ClusterDeploymentTypeMultiNodeReadReplicas)),
```

Comment:
This is breaking

Reply:
Because `deployment_type` can't be set in the API call for InfluxDB V3 clusters I am having to change the default to be dynamic (i.e. adding a `ModifyPlan` method to set the default for v2 clusters and otherwise leave as `null`).